test/api/v3/integration/user/auth/POST-register_local.test.js

```diff
@@ -71,6 +71,23 @@ describe('POST /user/auth/local/register', () => {
       await expect(getProperty('users', user._id, '_ABtests')).to.eventually.be.a('object');
     });
 
+    it('includes items awarded by default when creating a new user', async () => {
+      let username = generateRandomUserName();
+      let email = `${username}@example.com`;
+      let password = 'password';
+
+      let user = await api.post('/user/auth/local/register', {
+        username,
+        email,
+        password,
+        confirmPassword: password,
+      });
+
+      expect(user.items.quests).to.not.be.empty;
+      expect(user.purchased.background).to.not.be.empty;
```

Comment:
These checks should not just check for not empty, they should check for equality.